### DISCUSSION TITLE
進捗相談

### DISCUSSION THREAD

**Discussion by Ayato-kosaka** [Ayato-kosaka/react-itinerary#819] [General]
進捗相談チャンネル

**Comment by Ayato-kosaka** [comment]
Week 35

**Reply by Ayato-kosaka** [reply]
@Takapy1 
week34残と、week35の進捗報告します。
サムネイルと、フロントエンドテストで
両方結構課題出たけど、一旦物理的に作業できないから、
全体計画の方に移る！

一旦バッファは使わない方向で、
サムネイルと、フロントエンドテスト
をweek 36で片付けたいから、

week35 では、KPI達成方法、
全体スケジュール、リーンキャンバス更新を片付ける！


![image](https://github.com/Ayato-kosaka/spelieve/assets/53715839/018578dd-7c6e-4ccd-9fd6-59f0276b9d45)

**Reply by Ayato-kosaka** [reply]
なので、課題化したサムネイル・テストはスプリントバックログも後ろに回した！

**Reply by Takapy1** [reply]
@Ayato-kosaka 
おけー！
俺はやっと作業できるようになり、
ほぼ1週間分ずれたからバッファ使うことになると思う

**Comment by Ayato-kosaka** [comment]
Week 36

**Reply by Ayato-kosaka** [reply]
対面で会話予定
- フロントエンドテストは後回しにしたい
- Detox はもう無理だから落としたい
- Static Test の CIを作成も落としたいけど、コスパを考慮して要相談
-
